package.json

```diff
@@ -90,12 +90,13 @@
     "@bower_components/ovh-manager-webfont": "ovh-ux/ovh-manager-webfont#^1.0.1",
     "@bower_components/uri.js": "medialize/URI.js#~1.15.0",
     "@bower_components/validator-js": "chriso/validator.js#~3.37.0",
-    "ovh-ui-angular": "~2.15.0",
-    "ovh-ui-kit": "~2.15.0",
-    "popper.js": "^1.14.1"
+    "ovh-ui-angular": "^2.15.0",
+    "ovh-ui-kit": "^2.15.0",
+    "popper.js": "^1.14.1",
+    "serve": "^9.1.0"
```

Comment:
I think the changes in this file should be moved to another PR that explains what it does and how it improves the build.

Reply:
Furthermore I have no idea why `"serve"` is installed. Does this dependency is required/used?